skimage/skimage_lbp_simple.py

```diff
@@ -12,26 +12,25 @@
 import optuna
 
 import skimage.feature as ft
-from sklearn.datasets import fetch_olivetti_faces
+from sklearn.datasets import load_digits
 import sklearn.metrics
 from sklearn.metrics.pairwise import cosine_distances
 from sklearn.metrics.pairwise import euclidean_distances
 
 
 def load_data():
-    rng = np.random.RandomState(0)
-    dataset = fetch_olivetti_faces(shuffle=True, random_state=rng)
-    faces = dataset.images
+    dataset = load_digits()
+    digits = dataset.images.astype(np.uint8)
```

Comment:
[Notes] The value of digits is 0 to 16.

> This generates an input matrix of 8x8 where each element is an integer in the range 0..16.

https://github.com/scikit-learn/scikit-learn/blob/8c9c1f27b7e21201cfffb118934999025fd50cca/sklearn/datasets/_base.py#L834C2-L835
https://archive.ics.uci.edu/ml/datasets/Optical+Recognition+of+Handwritten+Digits

To avoid warning messages, specifying `np.uint8` seems reasonable to me.
https://github.com/scikit-image/scikit-image/blob/8787a5ea2af68d0d1c0abdc66fb0358b59d09fb1/skimage/feature/texture.py#L352-L357

Reply:
[Notes] The `faces` of `fetch_olivetti_faces` is floating-point type. So, changing the dataset seems reasonable instead of casting to int.

https://github.com/scikit-learn/scikit-learn/blob/8c9c1f27b7e21201cfffb118934999025fd50cca/sklearn/datasets/_olivetti_faces.py#LL130C2-L130C2